eclair-core/src/main/scala/fr/acinq/eclair/PluginParams.scala

```diff
@@ -51,7 +52,7 @@ trait CustomCommitmentsPlugin extends PluginParams {
    * expire. If your plugin defines non-standard HTLCs, and they need to be automatically failed, they should be
    * returned by this method.
    */
-  def getIncomingHtlcs: Seq[IncomingHtlc]
+  def getIncomingHtlcs(nodeParams: NodeParams)(implicit log: LoggingAdapter): Seq[IncomingHtlc]
```

Comment:
`getIncomingHtlcs` now takes `NodeParams` (because plugin sees `Kit` class too late in `onKit` method while `getIncomingHtlcs` is called at `Setup` phase) and `LoggingAdapter` (because `IncomingPacket.decrypt` is reused in plugin and it takes a `LoggingAdapter` implicitly).

Reply:
Then it makes sense to do the same for `getHtlcsRelayedOut` (add `nodeParams` and the `LoggingAdapter`), it will always be useful in some cases isn't it?

Reply:
Probably won't hurt, also I've decided to remove `implicit` part from log since that seems too scala-specific while plugins could be done in other jvm languages.